### ISSUE TITLE
[Question]: How to redirect back to cordova app?

### ISSUE THREAD

**Issue by dewebeloper** [damienbod/angular-auth-oidc-client#1367]
**What Version of the library are you using?**
12.0.3

**Question**
How can I redirect back to cordova app?
I'm following an article [Securing Cordova App](https://offering.solutions/blog/articles/2020/09/17/securing-a-cordova-app-implemented-with-angular-using-oidc-and-oauth2/#adding-the-authentication-and-callback-in-the-angular-app).

I was managed to redirect to login page, enter credentials and login user. All that was fine and after login IDS4 redirect me to Redirect page. There is signin-redirect.js and I can see RedirectUrl is 
`/connect/authorize/callback?client_id=MyClientId&redirect_uri=myredirecturi://&response_type=code&scope=openid profile email roles offline_access public blue_bid_api&nonce=8125c4d2000b41aa17aaee073b06886468Z9J2ZCq&state=37bd7fc7a735d03d00ec0fea27febb5a4ahKeeaCm&code_challenge=DT28IaK4JNXpEILx36jLzMnEqBFDEozi7mOe66Er0xw&code_challenge_method=S256`

But problem is, I'm stuck on this page. There is no redirection to my Cordova app?
I tested my custom url schema on way that I create href=myredirecturi:// on some html page and that link open my app.
Confusing thing is that same href doesn't work from that Redirection page where I'm stuck.
Is there anything else that I can do with this issue?

**Comment by dewebeloper** [comment]
Hi @FabianGosebrink do you maybe have somewhere on git source of your article [Securing Cordova App](https://offering.solutions/blog/articles/2020/09/17/securing-a-cordova-app-implemented-with-angular-using-oidc-and-oauth2/#adding-the-authentication-and-callback-in-the-angular-app)?

**Comment by dewebeloper** [comment]
After some attempts I managed to to see issue.
If I use 
```
 this.oidcSecurityService.authorizeWithPopUp().subscribe(({ isAuthenticated, userData, accessToken, errorMessage }) => {
      console.log(isAuthenticated);
      console.log(userData);
    });
```
and inAppBrowser I was able to see final redirect url. 
![InkedScreenshot_20220127-075137_LI](https://user-images.githubusercontent.com/3725912/151308154-27a39f83-c69c-498c-a0f2-fe33dfb585dc.jpg)

and that url is obvious wrong.
So, anybody has any idea, who can cause that http:// is added at the beginning of redirection uri?
Is issue on IdentityServer4, angular-auth-oidc-client or maybe in InAppBrowser?

Any idea how to I overcome this issue?
